libraries/lib-wave-track/WaveClip.h

```diff
@@ -471,10 +512,18 @@ class WAVE_TRACK_API WaveClip final :
       bool committed{ false };
    };
 
+   // Real-time durations, i.e., stretching the clip modifies these.
    double mSequenceOffset { 0 };
    double mTrimLeft{ 0 };
    double mTrimRight{ 0 };
 
+   // Used in GetStretchRatio which computes the factor, by which the sample
+   // interval is multiplied, to get a realtime duration.
+   double mClipStretchRatio = 1.;
+   std::optional<double> mRawAudioTempo;
+   std::optional<double> mProjectTempo;
+
+   // Sample rate of the raw audio, i.e., before stretching.
```

Comment:
Here you can simply use `//!` for a Doxygen comment on one member.

Reply:
Isn't it too much to have private members show up in Doxygen?

Reply:
All might help someone needing to understand the implementation, not just interface.

There is the notion of “representation invariant” for instance — invariant conditions involving private member variables.  Might as well follow conventions with `@invariant` There too